coffee_demo_moves:
  - arm1/kettle_init
  - gripper1/kettle_open
  - arm1/kettle_grab
  - gripper1/kettle_close
  - arm1/kettle_lift
  - arm1/kettle_move2dripper
  - arm1/kettle_pour
  - arm1/kettle_move2dripper
  - arm1/kettle_lift
  - arm1/kettle_grab
  - gripper1/kettle_open
  - arm1/kettle_init
  - gripper3/cup_open
  - arm3/cup_grab
  - gripper3/cup_close
  - arm3/cup_lift
  - arm3/cup_place
  - gripper3/cup_open
  - arm3/raise
  - arm3/dripper_init
  - gripper3/dripper_open
  - arm3/dripper_grab
  - gripper3/dripper_close
  - arm3/dripper_lift_small
  - arm3/dripper_pull2side
  - arm3/dripper_lift_full
  - arm3/dripper_move
  - arm3/dripper_pour
  - arm3/dripper_move
  - arm3/move_base
  - gripper3/dripper_open
arm1:
  kettle_init: [0.346, 0.010, 0.291, 1.57, 0.035, 0.029]
  kettle_grab: [0.480, 0.015, 0.270, 1.57, 0.122, 0.032]
  kettle_lift: [0.480, -0.015, 0.513, 1.57, -0.051, -0.025]
  kettle_move2dripper: [-0.105, -0.396, 0.568, 1.57, -0.041, -1.830]
  kettle_pour: [-0.100, -0.45, 0.555, 2.3, -0.024, -1.896]
gripper1:
  kettle_open: [0.5]
  kettle_close: [1.0]
arm3:
  cup_grab: [0.482, 0.0, 0.15, 0.0, 0.0, 0.0]
  cup_lift: [0.34, -0.38, 0.230, 0, 0, -0.859]
  cup_place: [0.34, -0.38, 0.15, 0, 0, -0.881]
  raise: [0.117, -0.239, 0.172, 0, 0.762, -1.114]  
  dripper_init: [0.044, -0.310, 0.197, 0.468, 0.036, -1.431]
  dripper_grab: [0.058, -0.373, 0.192, 0.467, -0.036, -1.412]
  dripper_lift_small: [0.058, -0.363, 0.200, 0.467, -0.036, -1.412]
  dripper_pull2side: [0.216, -0.289, 0.186, 0.287, -0.060, -0.928]
  dripper_lift_full: [0.230, -0.339, 0.344, 0.287, -0.085, -0.974]
  dripper_move: [0.439, -0.218, 0.365, 0.429, -0.034, -0.461]
  dripper_pour: [0.400, -0.250, 0.355, 1.20, -0.067, -0.7]
  move_base: [0.445, -0.143, 0.194, 0.290, -0.077, -0.312]
gripper3:
  dripper_open: [0.85]
  dripper_close: [1.09]
  cup_open: [0.0]
  cup_close: [0.56]